tools/serverpod_cli/lib/src/generator/open_api/open_api_generator.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:convert';
+
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:path/path.dart' as p;
+import 'open_api_definition.dart';
+
+///  Generates OpenAPI specifications for a Serverpod project.
+class OpenApiGenerator {
+  const OpenApiGenerator();
+
+  String getOpenApiSchema(
```

Comment:
Shouldn't this be a private method? Naming isn't great as it generates code. What sets it apart from the `generateOpenApiSchema` method?